webapplication/aoi/models.py

```diff
@@ -20,6 +25,8 @@ class AoI(models.Model):
     polygon = models.PolygonField(spatial_index=True, verbose_name='Polygon')
     createdat = models.DateTimeField(default=timezone.now)
     type = models.IntegerField(choices=AREA_TYPE_CHOICES, default=USER_DEFINED_TYPE)
+    available_dates = models.JSONField(blank=True, null=True, verbose_name='Available dates')
```

Comment:
it is better to make clear available dates of what

Reply:
updated